xilem_web/src/templated.rs

```diff
@@ -70,7 +71,6 @@ where
         // If this is the same value, or no rebuild was forced, there's no need to rebuild
         if core::mem::take(&mut view_state.dirty) || !Rc::ptr_eq(&self.0, &prev.0) {
```

Comment:
Why is this still needed?

Reply:
Good point/catch. It now uses `Rc<impl View>` as inner view, so yeah not needed anymore.